site/src/pages/_app.page.tsx

```diff
@@ -114,16 +115,7 @@ const MyApp = ({
           <ThemeProvider theme={theme}>
             <CssBaseline />
             <PageLayout>
-              <Head>
-                <title>
-                  Block Protocol – an open standard for data-driven blocks
-                </title>
-                <meta itemProp="name" content="Block Protocol" />
-                <meta
-                  itemProp="description"
-                  content="An open standard for data-driven blocks"
-                />
-              </Head>
+              <DefaultSeo {...defaultSeoConfig} />
```

Comment:
Removed the meta tags from `_document.page.tsx` and moved them here as [next-seo documentation suggests](https://github.com/garmeeh/next-seo#default-seo-configuration)